### PR TITLE
docs: observability new relic dashboard generator

### PR DESCRIPTION
**Pull request by sissbruecker**
Add instructions and tool for creating a preconfigured dashboard in New Relic.

### CODE REVIEW

File: articles/tools/observability/integrations/newrelic.asciidoc [hunk 1]
```diff
@@ -74,3 +74,19 @@ Now you will have a menu where you can look through the telemetry data from you
 |https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-metrics-explorer-page[Metrics]
 |Explore all reported metrics
 |===
+
+== Dashboard
+
+We provide a preconfigured dashboard for New Relic that shows several metrics such as request throughput, error rate or JVM stats.
```
Comment (caalador) [inline articles/tools/observability/integrations/newrelic.asciidoc:80]:
```suggestion
We provide a preconfigured dashboard for New Relic that contains several metrics such as request throughput, error rate and JVM stats.
```

File: articles/tools/observability/integrations/newrelic.asciidoc [hunk 2]
```diff
@@ -74,3 +74,19 @@ Now you will have a menu where you can look through the telemetry data from you
...
+
+To use the dashboard, enter your New Relic account ID below and then click on `Generate`.
+You can get your account ID from the New Relic URL in the browser.
```
Comment (caalador) [inline articles/tools/observability/integrations/newrelic.asciidoc:83]: Would this perhaps warrant being a note?
```suggestion

[NOTE]
You can get your account ID from the New Relic URL in the browser.
```

File: articles/tools/observability/integrations/newrelic.asciidoc [hunk 3]
```diff
@@ -74,3 +74,19 @@ Now you will have a menu where you can look through the telemetry data from you
...
+
+[source,html, role=render-only]
+----
+include::{root}/frontend/demo/tools/observability/newrelic/dashboard-generator.ts[render,tags=snippet,indent=0]
+----
+
+Copy the dashboard JSON from above, and then open the New Relic UI in the browser.
+Open the `Dashboards` view, and then click on `Import dashboard` in the top-right.
+Paste the JSON that you copied from above, and then click on `Import dashboard`.
```
Comment (caalador) [inline articles/tools/observability/integrations/newrelic.asciidoc:92]: Would this make sense being a list?
```suggestion
* Copy the dashboard JSON from above
* Open the New Relic UI in the browser.
* Open the `Dashboards` view, and then click on `Import dashboard` in the top-right.
* Paste the JSON that you copied from above, and then click on `Import dashboard`.
```

File: articles/tools/observability/integrations/newrelic.asciidoc (hunk 1 above)
Comment (sissbruecker) [reply articles/tools/observability/integrations/newrelic.asciidoc]: Done

File: articles/tools/observability/integrations/newrelic.asciidoc (hunk 2 above)
Comment (sissbruecker) [reply articles/tools/observability/integrations/newrelic.asciidoc:85]: Done

File: articles/tools/observability/integrations/newrelic.asciidoc (hunk 3 above)
Comment (sissbruecker) [reply articles/tools/observability/integrations/newrelic.asciidoc]: Done

**Review by caalador — APPROVED** [review]

**Pull request merged**